src/main/java/com/google/sps/data/Time.java

```diff
@@ -1,22 +1,43 @@
 package com.google.sps.data;
 
+import java.util.List;
 import java.util.Objects;
 
-public class Time implements Comparable<Time>{
+public class Time implements Comparable<Time> {
   private Long start;
   private Long end;
+  private String ownerID;
+  private List<String> availableAttendees;
+
+  public Time(Long start, Long end, List<String> availableAttendees) {
```

Comment:
Optional: I think Time is maybe not a good name anymore because you are including attendees now. Maybe make it clear that it is connected to an event and name it EventTime?